templates/house_view/house_create.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %} {% load static %}
+{% load crispy_forms_tags %}
+{% block content %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/house_create.css' %}"/>
+    <div id="house_create">
+        <div class="d-flex justify-content-center h-100">
+            <div class="card">
+                <div class="card-header">
+                    <h3>Add your House information</h3>
```

Comment:
I think that, because it is a title, All of the starting letters should be upper-case letters:
"Add Your House Information"

Reply:
Sure why not